css/definitions.json

```diff
@@ -1,69 +1,366 @@
 {
-  "groupList": {
-    "enum": [
-      "Basic Selectors",
-      "Combinators",
-      "Compositing and Blending",
-      "CSS Angles",
-      "CSS Animations",
-      "CSS Backgrounds and Borders",
-      "CSS Box Model",
-      "CSS Box Alignment",
-      "CSS Break",
-      "CSS Charsets",
-      "CSS Color",
-      "CSS Columns",
-      "CSS Conditional Rules",
-      "CSS Containment",
-      "CSS Counter Styles",
-      "CSS Device Adaptation",
-      "CSS Display",
-      "CSS Flexible Box Layout",
-      "CSS Flexible Lengths",
-      "CSS Fonts",
-      "CSS Fragmentation",
-      "CSS Frequencies",
-      "CSS Generated Content",
-      "CSS Grid Layout",
-      "CSS Images",
-      "CSS Inline",
-      "CSS Lengths",
-      "CSS Lists and Counters",
-      "CSS Logical Properties",
-      "CSS Masking",
-      "CSS Miscellaneous",
-      "CSS Motion",
-      "CSS Namespaces",
-      "CSS Pages",
-      "CSS Positioning",
-      "CSS Regions",
-      "CSS Resolutions",
-      "CSS Ruby",
-      "CSS Scroll Snap",
-      "CSS Shapes",
-      "CSS Speech",
-      "CSS Table",
-      "CSS Text",
-      "CSS Text Decoration",
-      "CSS Times",
-      "CSS Transforms",
-      "CSS Transitions",
-      "CSS Types",
-      "CSS Units",
-      "CSS Basic User Interface",
-      "CSS Variables",
-      "CSS Will Change",
-      "CSS Writing Modes",
-      "CSSOM View",
-      "Filter Effects",
-      "Media Queries",
-      "Microsoft Extensions",
-      "Mozilla Extensions",
-      "Pointer Events",
-      "Pseudo-classes",
-      "Pseudo-elements",
-      "Selectors",
-      "WebKit Extensions"
-    ]
+  "alsoApplyTo": {
+    "type": "array",
+    "minItems": 1,
+    "uniqueItems": true,
+    "items": {
+      "enum": [
+        "::first-letter",
+        "::first-line",
+        "::placeholder"
+      ]
+    }
+  },
+  "propertyList": {
+    "type": "array",
+    "minItems": 1,
+    "uniqueItems": true,
+    "items": {
+      "$ref": "properties#/names"
+    }
+  },
+  "enums": {
```

Comment:
Why we need to split types for paths `enums/names/*` and `enums/text/*`? I believe it can be a plain list of types, i.e. `groups` instead of `enums/names/groups` and `initial` instead of `enums/text/initial`

Reply:
The reason why I split them, was because I am cross-checking `enums/text` with `l10n/css`. So if the key is missing in l10n or l10n has keys which does not exists in this list, the build will fail. This way I found some unused l10n strings. I felt like it is a good idea to keep things tidy.

I was also considering on dumping the whole `enum/text` and simply pull the values in js directly from individual props in `*.json`. This would limit the overhead of adding the new values into `definitions.js`. What do you think?